Fix bugs on windows,'async:1: no such file or directory: ../external/async/async.zsh'

Fpointzero opened the pull request:
Modified the README.md file and provided a bat command to fix the error caused by creating soft links in Windows.
Fixes #2016

## Proposed Changes

  - Modified the README.md ,provide the bat command to fix bugs.

Fpointzero committed:
Modified the REAMME.md file and provided a bat command to fix the error caused by creating soft links in Windows.

Fpointzero committed:
Remove redundant content

jeffwidman reviewed (commented):
Can you remove the trailing whitespace at end of line?

jeffwidman commented:
Also, I don't use Windows, so it'd be great if someone who does could test this to get a second pair of 👀 before merging...

elkeis commented:
> Can you remove the trailing whitespace at end of line?

Sorry for saying from the nowhere, but IMHO, that trailing space highlight in git diff, especially for .md file, looks more like a GitHub Bug for me in 2k24.

Fpointzero committed:
Remove the trailing whitespace at end of line.

Fpointzero commented:
> Can you remove the trailing whitespace at end of line?

Maybe now is ok? I'm not sure. I use python script to remove.

Fpointzero committed:
fix bugs(to Change HomeDir to MSYS2's user home)

Fpointzero committed:
change / to \

oldboys92 commented:
`mklink` command works only with local admin rights. at least user should be aware that these are required.

Fpointzero committed:
fix:
- add a new way to solve `no such file or directory: ../external/async/async.zsh` when using windows with msys2
docs:
- move `example.bat` at 02.

Fpointzero commented:
@oldboys92 if using msys2, thereis a new way just set environment variable by `export MSYS=winsymlinks:lnk`.

Fpointzero committed:
docs: format README.md

Fpointzero committed:
Merge branch 'sorin-ionescu:master' into master

Fpointzero committed:
Merge branch 'sorin-ionescu:master' into master

Fpointzero committed:
fix: this version is not need administrator.

Fpointzero committed:
Merge branch 'sorin-ionescu:master' into master
